k8s/namespaces/civil/civil-service/prod.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: civil-service
+spec:
+  values:
+    java:
+      environment:
+        OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-prod2.internal:8443/openam/oauth2/realms/root/realms/hmcts
```

Comment:
do you need to override the idam api url for prod? idam-api.platform.hmcts.net ?

Reply:
it's OIDC issuer